_topic_maps/_topic_map.yml

```diff
@@ -3494,25 +3494,25 @@ Topics:
   Topics:
   - Name: Day 2 operations for telco core CNF clusters
     File: telco-day-2-welcome
-  - Name: Upgrading telco core CNF clusters
+  - Name: Upgrading OpenShift clusters
```

Comment:
{product-title} instead of OpenShift in all instances in this PR
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#openshift